internal/ls/hover.go

```diff
@@ -355,7 +355,7 @@ func getJSDocOrTag(node *ast.Node) *ast.Node {
 		return getMatchingJSDocTag(node.Parent, node.Name().Text(), isMatchingParameterTag)
 	case ast.IsTypeParameterDeclaration(node):
 		return getMatchingJSDocTag(node.Parent, node.Name().Text(), isMatchingTemplateTag)
-	case ast.IsVariableDeclaration(node) && core.FirstOrNil(node.Parent.AsVariableDeclarationList().Declarations.Nodes) == node:
+	case ast.IsVariableDeclaration(node) && ast.IsVariableDeclarationList(node.Parent) && core.FirstOrNil(node.Parent.AsVariableDeclarationList().Declarations.Nodes) == node:
```

Comment:
Given the test case is about a catch clause, I'm assuming this is a case where a variable declaration parent can be a declaration list or a catch clause.

Does this mean that we should actually just be handling both? I don't think we actually allow jsdoc in this position, so presumably not?